scheduler/scheduler.go

```diff
@@ -296,6 +296,10 @@ func parentCapacity(d distro.Distro, numNewParents, numCurrentParents, numCurren
 	if numNewParents+numCurrentParents > d.PoolSize {
 		numNewParents = d.MaxContainers - numCurrentParents
 	}
+	// if looking at a static docker provider, do not spawn any more parents
```

Comment:
I would move this to the top of the function since nothing else is relevant for static parents

Reply:
Actually this still would mean the `if numNewParents+numCurrentParents > d.PoolSize` block will still run before returning at the end. Should I just immediately return after each if statement instead?

Reply:
Or use `else if`s? I don't know if we use those often

Reply:
I would move this to the top and return 0 immediately. I think that's the simplest